bin/regressive_test.sh

```diff
@@ -14,6 +14,17 @@ fi
 
 ./recompile.sh
 . ./install_dbtoaster.sh
+# check if scala installed
+if hash scala 2>/dev/null; then
+    version=$(scala -version 2>&1 | awk -F " " '{print $5}') 
+    if [[ $version != "2.10"* ]]; then
```

Comment:
@khuevu this only accepts *exactly* version 2.10. Is there any reason for this? It seems to work fine for me with version 2.11